protostar/commands/init/project_creator/_project_creator.py

```diff
@@ -28,14 +28,20 @@ def copy_template(self, template_name: Literal["default"], project_root_path: Pa
         template_path = self.script_root / "templates" / template_name
         shutil.copytree(template_path, project_root_path)
 
-    def save_protostar_toml(self, project_root: Path, libs_path: Path) -> None:
+    def save_protostar_toml(self, project_root: Path) -> None:
         self.protostar_toml_writer.save(
             path=project_root / "protostar.toml",
             protostar_config=ProtostarConfigSection.get_default(self.version_manager),
-            protostar_project=ProtostarProjectSection(libs_relative_path=libs_path),
+            protostar_project=ProtostarProjectSection(
+                libs_relative_path=Path(self.default_lib_dirname),
+            ),
             protostar_contracts=ProtostarContractsSection.get_default(),
         )
 
+    @property
+    def default_lib_dirname(self) -> str:
+        return "lib"
```

Comment:
[Complexity] This property could be defined in the constructor.

Reply:
done